v1/proto/gribi_aft/enums/enums.pb.go

```diff
@@ -152,16 +152,17 @@ func (OpenconfigMplsTypesMplsLabelEnum) EnumDescriptor() ([]byte, []int) {
 type OpenconfigPacketMatchTypesIPPROTOCOL int32
 
 const (
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_UNSET   OpenconfigPacketMatchTypesIPPROTOCOL = 0
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_GRE  OpenconfigPacketMatchTypesIPPROTOCOL = 24050007
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_TCP  OpenconfigPacketMatchTypesIPPROTOCOL = 38721802
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_L2TP OpenconfigPacketMatchTypesIPPROTOCOL = 81903923
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_AUTH OpenconfigPacketMatchTypesIPPROTOCOL = 203904199
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_PIM  OpenconfigPacketMatchTypesIPPROTOCOL = 272904165
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_IGMP OpenconfigPacketMatchTypesIPPROTOCOL = 512271866
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_ICMP OpenconfigPacketMatchTypesIPPROTOCOL = 512860246
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_UDP  OpenconfigPacketMatchTypesIPPROTOCOL = 525100026
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_RSVP OpenconfigPacketMatchTypesIPPROTOCOL = 530070378
+	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_UNSET    OpenconfigPacketMatchTypesIPPROTOCOL = 0
+	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_GRE   OpenconfigPacketMatchTypesIPPROTOCOL = 24050007
+	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_TCP   OpenconfigPacketMatchTypesIPPROTOCOL = 38721802
```

Comment:
There's a white space change here different from what went in #27, is this expected?

Reply:
It looks like it's because of `IP_IN_IP`, so yes, I think this is likely reasonable.